circus-train-aws-sns/src/main/java/com/hotels/bdp/circustrain/aws/sns/event/SnsMessage.java

```diff
@@ -129,6 +130,10 @@ public String getReplicaMetastoreUris() {
     return modifiedPartitions;
   }
 
+  public void clearModifiedPartitions() {
+    modifiedPartitions.clear();
```

Comment:
field can be null

Reply:
Good catch, added null check and a tests for calling clearModifiedPartitions() with it being null and not.